daemon/network.go

```diff
@@ -372,7 +372,10 @@ func (daemon *Daemon) GetNetworkDriverList() []string {
 		}
 	}
 	// TODO : Replace this with proper libnetwork API
-	pluginList = append(pluginList, "overlay")
+	if !pluginMap["overlay"] {
+		pluginList = append(pluginList, "overlay")
+		pluginMap["overlay"] = true
```

Comment:
Perhaps move these two lines above the for-loop above, and just initialize pluginList, and pluginMap there;

```go
pluginList := []string{"overlay"}
pluginMap := map[string]bool{"overlay":true}
```